libs/angular-instantsearch/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "angular-instantsearch",
-  "version": "4.4.1",
+  "name": "@carnegie-learning/angular-instantsearch",
+  "version": "5.0.1",
   "peerDependencies": {
     "@angular/common": "^17.0.0",
     "@angular/core": "^17.0.0",
-    "algoliasearch": "4.10.3",
-    "algoliasearch-helper": "3.8.3",
+    "algoliasearch": "4.22.1",
+    "algoliasearch-helper": "3.16.3",
```

Comment:
Should we set these as version ranges instead of fixed versions?

Reply:
Definitely, that's a good idea. Should we use `^` here? Also, `data-access-cure` uses `algoliasearch@4.17.0` so would it have issues?

Should we set the version to `"algoliasearch": "^4.17.0"` instead?

Reply:
`^` is probably preferred.

You can either match `data-access-cure` or not care that you're enforcing a newer version.  I don't think it matters, since the version ranges intersect.

Reply:
Oh right, `data-access-cure` also have a range so we should be good there.